### PR TITLE
BaSP-MR-66: Add and edit form 

### PR DESCRIPTION
**Pull request by ticilicarzze**
## Description

This pull request implements features that let you add and edit a table with a form. 

## Checklist

- [x] Create add/edit form including necessary fields
- [x] On edit functionality should show previous info
- [x] Success message for confirmation edit and add actions
- [x] Success message for confirmation edit and add actions

### CODE REVIEW

**Review by FrancoLelli — APPROVED** [review]
Nice Job!!

**Review by CristianLotorto — CHANGES_REQUESTED** [review]
nice job! But you pushed .gitignore and package-lock.json :/. Lets fix that

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,5 @@ yarn-debug.log*
 yarn-error.log*
 
 .env
+package.json
```
Comment (NicoCagnina) [inline .gitignore:26]: we should not add this to the gitignore

File: .gitignore [hunk 2]
```diff
@@ -23,3 +23,5 @@ yarn-debug.log*
...
+package-lock.json
```
Comment (NicoCagnina) [inline .gitignore:27]: we should not add this to git ignore

File: src/Components/Admins/Table/table.module.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.container{
+  /* border: 1px solid black; */
```
Comment (NicoCagnina) [inline src/Components/Admins/Table/table.module.css:2]: remove this comment

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

**Review by iaracriscenti — APPROVED** [review]
🚀

**Comment by GleViDe** [comment]
### 2nd Testing Results
Tested locally on https://nico-megarocket-app-git-admin-mr-66add-edit-form-basp.vercel.app/admins
Test result: Passed

Great job!!

**Pull request merged**